resources/icons/call-no-return.svg

```diff
@@ -1,19 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
- <defs>
```

Comment:
Curious, are these images not being used?

Reply:
These were removed with a feature that "visualiz[ed] codeflow step level changes". Some of the rationale for removal:
* We did not get feedback it was a critical feature.
* The icons have an adverse effect on indentation.
* The icons non-standard and not thematically aligned with VS Code.
* Added complexity to the code flows feature.
* It was an experimental feature we and we had not gotten user feedback on it.

That said, please do take a look at the feature (it's still in the old branch). If you feel it's important, we can bring it back.